### PR TITLE
NE-407: operator/ingress: Add ELB connection idle timeout

### PR DESCRIPTION
**Pull request by Miciah**
* `operator/v1/types_ingress.go` (`AWSClassicLoadBalancerParameters`): Add `ConnectionIdleTimeoutSeconds` field.
* `operator/v1/0000_50_ingress-operator_00-ingresscontroller.crd.yaml:`
* `operator/v1/zz_generated.swagger_doc_generated.go:` Regenerate.

----

Related to https://github.com/openshift/enhancements/pull/461.

### CODE REVIEW

**Comment by Miciah** [comment]
Rebased.

**Comment by Miciah** [comment]
Updated to match the current version of the enhancement PR.

File: operator/v1/types_ingress.go [hunk 1]
```diff
@@ -518,6 +518,16 @@ const (
...
+	// connectionIdleTimeout specifies the maximum time period that a
+	// connection may be idle before the load balancer closes the
+	// connection.  The value must be parseable as a time duration value;
+	// see <https://pkg.go.dev/time#ParseDuration>.  The default value for
+	// this field is 60s.
+	//
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Format=duration
+	// +optional
+	ConnectionIdleTimeout *metav1.Duration `json:"connectionIdleTimeout"`
```
Comment (JoelSpeed) [inline operator/v1/types_ingress.go:530]: Does this need an omitempty?

**Review by JoelSpeed — COMMENTED** [review]
/lgtm

**Comment by Miciah** [comment]
```
Trying to pull registry.build02.ci.openshift.org/ci/managed-clonerefs@sha256:d4f56ce76ea8cacb4f39b1cf7abaf2de8ac64274759bb94dc6341ffbc3f297f1...
error: error creating buildah builder: initializing source docker://registry.build02.ci.openshift.org/ci/managed-clonerefs@sha256:d4f56ce76ea8cacb4f39b1cf7abaf2de8ac64274759bb94dc6341ffbc3f297f1: pinging container registry registry.build02.ci.openshift.org: Get "https://registry.build02.ci.openshift.org/v2/": dial tcp: i/o timeout
error while getting build logs, could not retrieve build pod ci-op-pg9kfbhj/src-build : pod "src-build" not found[36mINFO[0m[2022-04-12T13:23:50Z] Ran for 1m18s                                
```
/test verify-deps

**Comment by deads2k** [comment]
/approve

/hold

hold pending link to why we cannot use non-pointer.

**Pull request merged**